changelog/v0.10.18/ready.yaml

```diff
@@ -0,0 +1,3 @@
+changelog:
+  - type: NON_USER_FACING
```

Comment:
definitely user facing, I'd consider it a new feature

Reply:
i guess it depends who is the user. i can change